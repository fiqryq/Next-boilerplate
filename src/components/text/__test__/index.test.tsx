import { render } from '@testing-library/react'
import { Text } from '../index'

const describeComponents = 'render text components'
describe(describeComponents, () => {
    test('should render correctly', () => {
        const { container } = render(<Text as='h1'>Hello World</Text>)
        expect(container).toMatchSnapshot()
    })
})
