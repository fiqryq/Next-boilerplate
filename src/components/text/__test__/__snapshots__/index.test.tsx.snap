// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render text components should render correctly 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
